repl/repl.go

```diff
@@ -408,6 +421,11 @@ func (r *REPL) cmdTrace() error {
 	return nil
 }
 
+func (r *REPL) cmdMetrics() error {
+	r.metricsEnabled = !r.metricsEnabled
```

Comment:
Nit: I'd be tempted to add a `Clear()` function to the metrics interface. This way we don't need a separate value to track whether metrics tracking is enabled. E.g., this would change to `r.metrics = metrics.New()` and then the REPL would Clear the metrics each time OneShot is called. Just a thought.